client/app/scripts/reducers/root.js

```diff
@@ -70,7 +71,7 @@ export const initialState = makeMap({
   updatePausedAt: null, // Date
   version: '...',
   versionUpdate: null,
-  websocketClosed: true,
+  websocketClosed: false,
```

Comment:
I guess... I read it is "it has been closed", like, past tense. I guess it could be present too though.

We interpret it as being a past tense thing in status.js and show a "Trying to reconnect..." briefly before we've established the ws during init.